src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonerSearchByBookingIdsResourceTest.kt

```diff
@@ -8,11 +8,13 @@ import org.awaitility.kotlin.matches
 import org.awaitility.kotlin.untilCallTo
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import org.springframework.test.context.TestPropertySource
 import org.springframework.web.reactive.function.BodyInserters
 import uk.gov.justice.digital.hmpps.prisonersearch.QueueIntegrationTest
 import uk.gov.justice.digital.hmpps.prisonersearch.services.PrisonerListCriteria.BookingIds
 import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.OffenderBooking
 
+@TestPropertySource(properties = ["index.page-size=12"])
```

Comment:
Fixes old bug which caused 4 * number of messages and hence timed out when tests ran locally.